### PR TITLE
Invite our community to Community Hackathon 2023

### PR DESCRIPTION
**Pull request by itrich**

### CODE REVIEW

File: _i18n/de/_posts/blog/2023-01-27-hackathon-2023.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title: "Sovereign Cloud Stack Community-Hackathon: Die Community schlägt zurück"
```
Comment (fkr) [inline _i18n/de/_posts/blog/2023-01-27-hackathon-2023.md:3]: I'd actually prefer a wording that is less aggressive

File: _i18n/de/_posts/blog/2023-01-27-hackathon-2023.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+## Verpflegung
+
+Zwischen 17:00 und 18:00 Uhr werden wir eine längere Pause einlegen, bei der es auch Pizza geben wird. Auf ein veganes und vegetarisches Angebot werden wir achten. Solltest du eine Unverträglichkeit haben, lass uns dies bitte vorab wissen!
+
+## Datenschutz und Hinweise
+
+Die Sicherheitsanforderungen des Standorts verlangen die persönliche Registrierung durch einen gültigen Personalausweis oder Reisepass. Die noris Network AG wird diese für die Dauer der Aufbewahrungsfrist speichern und im Falle einer Auditierung verarbeiten müssen.
+
+Darüber hinaus werden wir im Verlauf des Tages Fotos oder Videos von dem Event machen. Solltest du damit nicht einverstanden sein, weise uns bitte während deines Check-ins darauf hin. Du erhältst in diesem Fall einen "No camera" Aufkleber von uns, den du sichtbar auf deinem Namensschild anbringst. Alle Fotos werden vor Veröffentlichung in einem geschützten Bereich bereitgestellt, sodass ihr in jedem Fall euer Veto geben könnt.
```
Comment (fkr) [inline _i18n/de/_posts/blog/2023-01-27-hackathon-2023.md:75]: very, very good! I think this is very sensitive and this is an awesome way to approach that! Good learning from the first hackathon. Thanks @itrich!

File: _i18n/de/_posts/blog/2023-01-27-hackathon-2023.md (hunk 1 above)
Comment (itrich) [reply _i18n/de/_posts/blog/2023-01-27-hackathon-2023.md:3]: It's a reference to Star Wars, but feel free to propose an alternative blog post title!

Comment (fkr) [reply _i18n/de/_posts/blog/2023-01-27-hackathon-2023.md:3]: (and yes, I understand the intention with the Star Wars reference, it just doesn't work nicely imho on german).
Feel free to ignore my comment.

**Review by fkr — APPROVED** [review]
Awesome post!

File: _i18n/de/_posts/blog/2023-01-27-hackathon-2023.md (hunk 1 above)
Comment (itrich) [reply _i18n/de/_posts/blog/2023-01-27-hackathon-2023.md]: I've changed the title as you're right. We still can use the title at some other place 🤫

Comment (fkr) [reply _i18n/de/_posts/blog/2023-01-27-hackathon-2023.md]: Imho we can just use the english variant for the german title. But as I wrote, feel free to ignore.

Comment (itrich) [reply _i18n/de/_posts/blog/2023-01-27-hackathon-2023.md]: I'm confident with the latest change. Would you mind reviewing the post again?

**Pull request merged**